src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+			<li>
+				Added the ResetPassword security identifier that allows a user to reset another user’s passwords.
+			</li>
+		</ul>
+		<h2>Mg-Api-Js</h2>
+		<p>We've corrected the following errors:</p>
+		<ul>
+			<li>
+				Fixed UnhandledRejection exceptions.
+			</li>
+			<li>
+				Improved exception messages to include the type from the JSON-RPC error.
+			</li>
+			<li>
+				Fixed InvalidUserException retry authentication.
+			</li>
+			<li>
+				Updated CallBackError to be invoked as the last operation in the promise field.
+			</li>
+		</ul>
+		<h2>GetFeed{'<'}FuelTaxDetail{'>'}</h2>
+		<ul>
+			<li>
+				Fixed issues with results sorting that led to missing results.
+			</li>
+		</ul>
+		<h2>Nuget</h2>
+		<ul>
+			<li>
+				Enabled brotli compression in Nuget package API.cs requests.
+			</li>
+			<li>
+				<s>Many ObjectModel classes are marked as “sealed”. Marking a class as “sealed” prevents tampering of important classes that can compromise security or impact performance.</s>
+			</li>
+		</ul>
+		<InformationalBox>
+			<p><b>!IMPORTANT:</b> Updating your application to version 10.0 may not work if an ObjectModel class is inherited.</p>
+		</InformationalBox>
+		<h2>TextMessage</h2>
+		<ul>
+			<li>
+				Added PropertySelector (Beta) support for the TextMessage type.
+			</li>
+		</ul>
+		<h2>IoxAddOn</h2>
+		<ul>
+			<li>
+				Added PropertySelector (Beta) support for the IoxAddOn type.
+			</li>
+		</ul>
+		<h2>IoxAddOnStatus</h2>
+		<ul>
+			<li>Added PropertySelector (Beta) support for the IoxAddOnStatus type.</li>
+		</ul>
+		<h2>DriverChange</h2>
+		<ul>
+			<li>
+				Updated the description of DriverChange DateTime to notify users that the DateTime must not be in the future when adding a DriverChange through API.
+			</li>
+		</ul>
+		<h2>DriverWhiteListContent</h2>
+		<ul>
+			<li>
+				DriverWhiteListContent is replaced by <b>DriverAuthListContent</b>
+			</li>
+			<li>
+				Prior to the 10.0 release, DriverWhiteListContent and DriverAuthListContent were both available to allow users to adjust to the transition.
+			</li>
+		</ul>
+		<h2>Condition And ConditionType</h2>
+		<ul>
+			<li>
+				Added the Group exception rule condition. This separates conditions in a single rule when different groups require different conditions.
+			</li>
+		</ul>
+		<h2>Geotab Developers</h2>
+		<ul>
+			<li>
+				Added a section to the <a href="https://geotab.github.io/sdk/software/guides/developing-addins/">Developing Add-ins</a> page suggesting to use the generator-addin repository.
+			</li>
+		</ul>
+		<h2>SDK Samples</h2>
+		<ul>
+			<li>
+				Added JavaScript samples to refresh the user interface.
+			</li>
+		</ul>
+		<h2>SDK Add-In Samples</h2>
+		<ul>
+			<li>Added Add-In samples for the AddInData API to the <a href="https://github.com/Geotab/sdk-addin-samples">Geotab/sdk-addin-samples</a> Github repository.</li>
```

Comment:
<a> tags that point outside of the site should have target="_blank" rel="noopener noreferrer" attributes